ontology/yaml/resources/HVAC/entity_types/AHU.yaml

```diff
@@ -2667,6 +2667,55 @@ AHU_BYPSSPC_DX3SC_ECON_HT2SC_SFSS_SFVSC:
   - SFSS
   - SFVSC
 
+#AU-SY-ODI types  
+AHU_DFFC_ZTC_CHWZTC_HWZTC_STM_RTM_CO2C_FDPM_AHAC:
+  description: "Single zone air handling unit with CO2 override and after hours override"
+  is_canonical: true
+  implements:
+  - AHU
+  - DFFC
+  - ZTC
+  - CHWZTC
+  - HWZTC
+  - STM
```

Comment:
you implement **discharge_** for fans (DFFC), so this would be incorrect; should be DTM

Reply:
Got it. I think we now understand the discharge/supply difference, changes made.